stubs/extensions/ffi.phpstub

```diff
@@ -0,0 +1,451 @@
+         * @since 8.1
+         */
+        public const TYPE_UINT32 = 8;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_SINT32 = 9;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_UINT64 = 10;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_SINT64 = 11;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_ENUM = 12;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_BOOL = 13;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_CHAR = 14;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_POINTER = 15;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_FUNC = 16;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_ARRAY = 17;
+
+        /**
+         * @since 8.1
+         */
+        public const TYPE_STRUCT = 18;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_CONST = 1;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_INCOMPLETE_TAG = 2;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_VARIADIC = 4;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_INCOMPLETE_ARRAY = 8;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_VLA = 16;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_UNION = 32;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_PACKED = 64;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_MS_STRUCT = 128;
+
+        /**
+         * @since 8.1
+         */
+        public const ATTR_GCC_STRUCT = 256;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_DEFAULT = 0;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_CDECL = 1;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_FASTCALL = 2;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_THISCALL = 3;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_STDCALL = 4;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_PASCAL = 5;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_REGISTER = 6;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_MS = 7;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_SYSV = 8;
+
+        /**
+         * @since 8.1
+         */
+        public const ABI_VECTORCALL = 9;
+
+        public function getName(): string
+        {
+        }
+
+        /**
+         * @since 8.1.0
+         * @return self::TYPE_*
+         */
+        public function getKind(): int
+        {
+        }
+
+        /**
+         * @since 8.1.0
+         * @return 0|positive-int
```

Comment:
Just FYI, `0|positive-int` can now be annoted as `int<0, max>`. Don't feel obligated to change that, it'll be transparent in stubs anyway

Reply:
Thanks for the reminder, I had forgotten that.